components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8680,20 +8692,27 @@ def isGreyscaleRenderingModel(self):
 
     @assert_re()
     def setReverseIntensity(self, channelIndex, reverse=True):
+        """Deprecated in 5.4.0. Use setChannelInverted()."""
+        warnings.warn("Deprecated in 5.4.0. Use setChannelInverted()",
+                      DeprecationWarning)
```

Comment:
@jburel Warning was already added here, but it was the line below Josh's comment so it didn't register.

Reply:
thanks